server.js

```diff
@@ -0,0 +1,12 @@
+var express = require('express');
+var bodyParser = require('body-parser');
```

Comment:
Do we need to use the Body Parser with the express app (i.e. app.use(bodyParser) or something like that?